Week3/fuel.py

```diff
@@ -0,0 +1,22 @@
+
+def main():
+    gauge = fuel_tank()
+    if gauge <= 1:
+        print("E")
+    elif gauge >= 99:
+        print("F")
+    else:
+        print(gauge,"%")
+
+def fuel_tank():
+    while True:
+        fuel = input("Fraction: ")
+        try:
+            numerator, denominator = fuel.split("/")
```

Comment:
at this point, `numerator` and  `denominator` are `string` values

Reply:
done